frontend/src/metabase/query_builder/components/filters/modals/BulkFilterList/BulkFilterList.tsx

```diff
@@ -99,7 +94,7 @@ const BulkFilterListItem = ({
       <ListRowLabel data-testid="dimension-filter-label">
         {dimension.displayName()}
       </ListRowLabel>
-      <ListRowContent>
+      <>
```

Comment:
one thing that's not quite apparent here is how many children would this fragment has. If we just glace without thinking much we might say there could be 2 children but wait! Actually, only one of that clauses is rendered at a time. Maybe using the ternary or abstracting it to be another component might make it more obvious?
What do you think?

Reply:
I actually cleared up this code to fix some bugs in a downstream branch: 

you make a good point however, about the multiple components potentially causing issues with multiple filters on the same field:

![Screen Shot 2022-06-21 at 8 58 37 AM](https://user-images.githubusercontent.com/30528226/174832149-e7e98b5c-49e5-4da7-b75a-86cb979f61ad.png)

I'll need to play around with this a bit more